### PR TITLE
Add classification metrics to classification trainer 

### PR DESCRIPTION
**Pull request by talhaanwarch**
As in PR #25, user now can choose different classification metrics. Either 1 or many.
```
trainer = ClassificationTrainer(model=model, lr=1e-3, weight_decay=1e-4, device_ids=[0],
                               metrics=['accuracy','recall'])
```
If user did not defined any metric                                
```trainer = ClassificationTrainer(model=model, lr=1e-3, weight_decay=1e-4, device_ids=[0])```
then accuracy will be default metric.

Further i think creating a utils.py for each section, makes the main file less congested. Thats why I created that.

### CODE REVIEW

**Comment by tczhangzhi** [comment]
This proposal is absolutely reasonable, and their implementation seems to make sense. Would you like to add some test cases? I want to merge this PR, with the test cases added.

Or I can also supplement some test cases and documents on your branch.

**Comment by talhaanwarch** [comment]
If I am not wrong, i can add a test case here?
https://github.com/torcheeg/torcheeg/blob/954404aa83b5fa03b85840d0bc97a3ecd340e389/test/trainers/test_classification_trainer.py#L52

**Comment by talhaanwarch** [comment]
If I am not wrong, i can add a test case here?
https://github.com/torcheeg/torcheeg/blob/954404aa83b5fa03b85840d0bc97a3ecd340e389/test/trainers/test_classification_trainer.py#L52

**Comment by tczhangzhi** [comment]
Yes, I think so.

Maybe I should refer to PyTorch Lightning later to see how they test the trainer. However, for this PR, I think the test case you gave is feasible.

File: torcheeg/trainers/utils.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            'accuracy': torchmetrics.Accuracy(task='multiclass', num_classes=num_classes, top_k=1).to(device),
+            'precision': torchmetrics.Precision(task='multiclass', average='macro', num_classes=num_classes).to(device),
+            'recall': torchmetrics.Recall(task='multiclass', average='macro', num_classes=num_classes).to(device),
+            'f1score': torchmetrics.F1Score(task='multiclass', average='macro', num_classes=num_classes).to(device)
+        }
+    
+    else:
+        metric_list = ['accuracy']
+        metric_dict = {
+            'accuracy': torchmetrics.Accuracy(task='multiclass', num_classes=num_classes, top_k=1).to(device)}
```
Comment (tczhangzhi) [inline torcheeg/trainers/utils.py:22]: Thanks for your contribution, I guess the last thing is to check your package dependencies, e.g. `task='multiclass'` requires `torchmetrics>=0.10.0`. So we need to modify the corresponding content in `meta.yaml` and `setup.py`.

Even better if you can format the code with yepf.

**Comment by tczhangzhi** [comment]
Thanks for your contribution! I think this PR is ready to be merged.
